src/runtime/runtime.cs

```diff
@@ -354,6 +354,23 @@ internal static void Shutdown()
             Exceptions.Shutdown();
             ImportHook.Shutdown();
             Py_Finalize();
+
```

Comment:
Will test on Mac before merging

Reply:
If it fails then just document it as broken on mac.